demos/15-callbacks.html

```diff
@@ -19,8 +19,8 @@ <h2 class="sub-heading">EasyDropDown Demo</h2>
             <h1 class="heading">15. Callbacks</h1>
 
             <div class="demo-container">
-                <select id="test">
-                    <option value="" data-placeholder>Select</option>
+                <select id="test" required>
```

Comment:
Can you revert the changes to this demo file? I'm guessing you didn't intend to commit them?

Reply:
Actually i did this in purpose to demonstrate validation on close callback. (line 56) 
shall i revert the whole file?

Reply:
Ah i see - i was confused because the callbacks demo doesn't really to validation, so wondered if you'd just added there for your own testing. I think it will be better if we create a new demo for programmatic validation - but I'm happy to do that :) So let's revert this.

Reply:
sounds good. thanks.